starlite/openapi/parameters.py

```diff
@@ -115,6 +116,70 @@ def create_parameter(
     )
 
 
+def get_recursive_handler_parameters(
+    field_name: str,
+    model_field: "ModelField",
+    dependencies: Dict[str, "Provide"],
+    route_handler: "BaseRouteHandler",
+    path_parameters: List[Dict[str, Any]],
+    generate_examples: bool,
+) -> List[Parameter]:
+    """Create and return parameters for a handler. If the provided field is not a dependency,
+    a normal parameter is created and returned as a list, otherwise a recursive call
+    may be triggered by the parent call function"""
```

Comment:
> otherwise a recursive call may be triggered by the parent call function

I think this is a bit misleading as it is this function that triggers the recursive call. How about:
```
"""... otherwise `create_parameter_for_handler()` is called to generate parameters for the dependency."""
```
Something along those lines?

Reply:
you're absolutely right